### PR TITLE
His Grace

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Credit to Goonstation for this, as this is where it's very very heavily based off of. Implements "His Grace".

His Grace is an artistic toolbox that is embued with unholy powers; the first person to open the toolbox is blessed (or cursed) with the toolbox's power: Memetic Kill Agent. As the toolbox's follower, you are granted resistance to stuns and healing abilities as long as you have the toolbox on you. If you don't have it on you and you're too far away, you'll start to take damage.

What does His Grace want? Human souls, of course. You can feed unconscious/weakened/restrained people to the toolbox, which makes it even stronger...that said, this comes at a price. As His Grace grows stronger and stronger, so does his desire for human souls; if you fail to feed His Grace in time, he will turn on his follower and consume them, instead while His Grace will lose all of its previous power.

Destroying the toolbox will eject everyone it has consumed and cure the person of their condition "Memetic Kill Agent".

This is a whopping 20 TC item available *only* to the chaplain. Additionally, due to the murdery nature of the item, it is only available if the Chaplain specifically has the "hijack" objective. It is not available in surplus crates---being a traitor Chaplain with hijack is the literal only way this is available.

For those of you who already know how it works---things I changed for Para:
- Cannot feed it anything but human mobs that have a mind+ckey (no cheese farming)
- Mobs are not deleted when ate; they're stored in the toolbox and later ejected, when the toolbox is destroyed
- No bullshit examine-chaining of the disease associated with the toolbox or its wielder; only the first person to attempt to open the toolbox acquires the disease; once it's activated, no one else can re-activate it

Related Coder Tidbits
- Adds a system whereby specific items can be disallowed if the person using the uplink doesn't have hijack

pics:

img

:cl: Fox McCloud
Add: Adds in His Grace, a hijack-only traitor item for the Chaplain
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
This is hilarious. :+1:

File: code/game/objects/items/weapons/storage/artistic_toolbox.dm [hunk 1]
```diff
@@ -134,7 +134,7 @@
 		var/status = CANSTUN | CANWEAKEN | CANPARALYSE
 		affected_mob.status_flags &= ~status
 		affected_mob.dizziness = max(0, affected_mob.dizziness-10)
-		affected_mob.drowsyness = max(0, affected_mob:drowsyness-10)
+		affected_mob.drowsyness = max(0, affected_mob.drowsyness-10)
```
Comment (monster860) [inline code/game/objects/items/weapons/storage/artistic_toolbox.dm]: :eyes:

**Comment by Necaladun** [comment]
Wow, love the implementation of it. Hijack-only items has a lot of potential for other stuff too.

**Comment by CrAzYPiLoT-SS13** [comment]
Do I sense some scp kill agents? :+1:

**Comment by ProperPants** [comment]
>the first person to open the toolbox is blessed (or cursed) with the toolbox's power

This implies anyone can pick it up, but if it's a chaplain-only traitor item, wouldn't it always be the chaplain who picks it up?

Also, I don't think it should be a unique color. Then, people might actively hunt it to become the meme-lord. Granted, nobody might ever pick it up if it were the same color as other toolboxes.

**Comment by Fox-McCloud** [comment]
@ProperPants 

I don't want to rule this out for adminbus/events, which is why I didn't do a chaplain specific check for it---maybe admins want to do highland with everyone having these toolboxes instead of claymores. Who knows!

And...it's an incredibly powerful item; it needs to be visually distinctive so people know what's coming---like a double-bladed energy sword or chainsaw.

**Comment by taukausanake** [comment]
But then as a traitor how are you to hide it unless you're expected to go on a toolbox vore spree. Also wouldn't this eventually kill the user if they cannot feed the toolbox anymore? Let's say they get everyone on the station and call the shuttle. How long until His Grace decides to eat the user?
Another question because I don't Goon, how does one destroy the toolbox?

**Comment by Fox-McCloud** [comment]
You're not expected to hide it, that's the whole point. Just like a chainsaw. It's a risk the user takes on when he purchases it and activates it.

You destroy the toolbox by putting it in the recycler, an autolathe, acid, explosions, or how you you destroy any other object.

**Pull request merged**